### PR TITLE
Added button to rerun pipelines

### PR DESCRIPTION
**Pull request by dersvenhesse**
In my experience when using the gitlab-monitor a frequent use-case is to check for failed pipelines and retry them. Therefore I added a new button to rerun a pipeline completely (by starting a new pipeline run).

The button can be enabled when needed via the option `showRerunButton` and is disabled per default. To use this option, the provided token needs write access.

Furthermore, I added some slight tweaks:
* updated CSS for labels (introduced in #200)
* reordered default config to match order in description of configuration options
* added link to configuration page to learn about the configuration options
* removed unused option `fetchCountProjects` from description

Let me know what you think.

### CODE REVIEW

File: src/components/pipeline-view.vue [hunk 1]
```diff
@@ -194,6 +200,15 @@
...
       },
+      async rerunPipeline() {
+        let path;
+        if (!this.pipeline.ref.includes('merge-request')) {
+          path = `/projects/${this.project.id}/pipeline?ref=${this.pipeline.ref}`;
+        }
+        else {
+          path = `/projects/${this.project.id}/merge_requests/${this.pipeline.ref.match(/\d+/)}/pipelines`;
+        }
+        await this.$api(path, {}, {}, 'POST');},
```
Comment (timoschwarzer) [inline src/components/pipeline-view.vue:211]:
```suggestion
      async rerunPipeline() {
        await this.$api(
          !this.pipeline.ref.includes('merge-request')
            ? `/projects/${this.project.id}/pipeline?ref=${this.pipeline.ref}`
            : `/projects/${this.project.id}/merge_requests/${this.pipeline.ref.match(/\d+/)}/pipelines`,
          {}, {}, 'POST',
        );
      },
```

**Review by timoschwarzer — APPROVED** [review]

**Comment by timoschwarzer** [comment]
Thank you very much! Also for the additional chore :smile:

**Comment by dersvenhesse** [comment]
You're welcome. Do you mind deploying the recent changes to your server as well?

**Comment by timoschwarzer** [comment]
@dersvenhesse Done :)

**Pull request merged**